docs/cn/bthread_or_not.md

```diff
@@ -50,6 +50,19 @@ bool search() {
   return true;
 }
 ```
+
+或者使用更简洁的C++11风格接口，它具有与C++11 std::thread相同的语义：
+```c++
+void search() {
+    bthread::BThread th1(part1, part1_arg1, std::ref(part1_arg2)); // use functions & functors
```

Comment:
Since we already have the namespace("bthread"), the class name "BThread" feels redundant, "bthread::Thread" should be clear to users.